packages/Python/modern_robotics/core.py

```diff
@@ -158,6 +158,7 @@ def MatrixLog3(R):
                   [ 1.20919958,           0, -1.20919958],
                   [-1.20919958,  1.20919958,           0]])
     """
+	R = ProjectToSO3(R)
```

Comment:
This added line is currently indented with a tab character instead of 4 spaces like the rest of the file. May cause problems on Python 3, and is inconsistent.